### PR TITLE
Registration form with antd

### PR DESCRIPTION
**Pull request by ojo88**
Created a basic registration form to add alumni. Used AntD to make it look better. Still needs some work but is functional.

### CODE REVIEW

File: src/alumni/form.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+}
+
+
+function addToAlumniCollection() {
+    //This function will post the form data into the okcoders db
+}
+
+export default Form;
+
```
Comment (zmays) [inline src/alumni/form.js]: this sucks you suck

File: src/App.js [hunk 2]
```diff
@@ -4,15 +4,16 @@ import './App.css';
...
 import { Alumni } from "./alumni/alumni.component.js"
-
+import CustomForm from './alumni/form.js';
 
 class App extends Component {
   render() {
     return (
       <>
-        <Router>
+        {<Router>
```
Comment (zmays) [inline src/App.js]: why did you wrap this with `{`?

File: src/App.js [hunk 3]
```diff
@@ -4,15 +4,16 @@ import './App.css';
 import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import "antd/dist/antd.css";
 import { Alumni } from "./alumni/alumni.component.js"
-
+import CustomForm from './alumni/form.js';
```
Comment (zmays) [inline src/App.js]: Instead of custom form, perhaps call it something like AddAlumniForm

File: src/alumni/form.js [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+    const [email, setEmail] = useState('');
+    const [birthday, setBirthday] = useState('');
+    const [gitHub, setGitHub] = useState('');
+    const [linkedIn, setLinkedIn] = useState('');
+    const [bio, setBio] = useState('');
+
+    const { TextArea } = Input;
+    const { Header, Content, Footer } = Layout;
+
+    function clearFields() {
```
Comment (zmays) [inline src/alumni/form.js]: Nice

File: src/alumni/form.js [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+            lastName: lastName,
+            email: email,
+            github: gitHub,
+            linkedin: linkedIn,
+            bio: bio,
+            birthday: birthday
+        }
+
+
+        Axios.post('http://localhost:9000/alumni', { newAlumni })
```
Comment (zmays) [inline src/alumni/form.js]: get the url from the config

Comment (zmays) [reply src/alumni/form.js]: Also make sure to handle any errors

File: src/App.js (hunk 2 above)
Comment (ojo88) [reply src/App.js]: not sure how that got there. It's been removed.

File: src/alumni/form.js (hunk 5 above)
Comment (ojo88) [reply src/alumni/form.js]: got url form config

File: src/App.js (hunk 3 above)
Comment (ojo88) [reply src/App.js]: changed name to AddAlumniForm

File: src/alumni/form.js (hunk 5 above)
Comment (ojo88) [reply src/alumni/form.js]: forgot to handle errors

Comment (ojo88) [reply src/alumni/form.js]: added some error handling

**Pull request merged**